tests/e2e/cjs/index.test.js

```diff
@@ -359,14 +361,73 @@ describe('ANTRegistry', async () => {
 });
 
 describe('ANT', async () => {
+  const processId = 'YcxE5IbqZYK72H64ELoysxiJ-0wb36deYPv55wgl8xo';
   it('should be able to create ANTWriteable with valid signers', async () => {
     for (const signer of signers) {
       const ant = ANT.init({
-        processId: 'aWI_dq1JH7facsulLuas1X3l5dkKuWtixcZDYMw9mpg',
+        processId,
         signer,
+        strict: true,
+        ao: aoClient,
+      });
+      const strictAnt = ANT.init({
+        processId,
+        signer,
+        strict: true,
+        ao: aoClient,
       });
 
       assert(ant instanceof AoANTWriteable);
+      assert(strictAnt instanceof AoANTWriteable);
     }
   });
+
+  it(`should be able to read methods when strict is true`, async () => {
+    const strict = true;
+    const ant = ANT.init({
+      processId,
+      ao: aoClient,
+    });
+    const state = await ant.getState({ strict });
+    assert(state);
+    const info = await ant.getInfo({ strict });
+    assert(info);
+    const balance = await ant.getBalance(
+      { address: '7waR8v4STuwPnTck1zFVkQqJh5K9q9Zik4Y5-5dV7nk' },
+      { strict },
+    );
+    assert(balance);
+    const balances = await ant.getBalances({ strict });
+    assert(balances);
+    const controllers = await ant.getControllers({ strict });
+    assert(controllers);
+    const records = await ant.getRecords({ strict });
+    assert(records);
+    const record = await ant.getRecord({ undername: 'ardrive' }, { strict });
+    assert(record);
+  });
+  it(`should be able to read methods when strict is false`, async () => {
+    const strict = false;
+    const ant = ANT.init({
+      processId,
+      ao: aoClient,
+    });
+    const state = await ant.getState({ strict });
+    assert(state);
+    const info = await ant.getInfo({ strict });
+    assert(info);
+    const balance = await ant.getBalance(
+      { address: '7waR8v4STuwPnTck1zFVkQqJh5K9q9Zik4Y5-5dV7nk' },
+      { strict },
+    );
+    assert(balance);
+    const balances = await ant.getBalances({ strict });
+    assert(balances);
+    const controllers = await ant.getControllers({ strict });
+    assert(controllers);
+    const records = await ant.getRecords({ strict });
+    assert(records);
+    const record = await ant.getRecord({ undername: 'ardrive' }, { strict });
+    assert(record);
+  });
```

Comment:
nit, move these to just the esm tests - we have the coverage we need for CJS (really just validating the client imports and instantion works) - we can focus the full suite of integration tests on the `esm` tests without duplicating